plugins/cache_promote/configs.cc

```diff
@@ -99,6 +99,10 @@ PromotionConfig::factory(int argc, char *argv[])
     }
   }
 
+  if (!_policy) {
```

Comment:
compare to nullptr instead of just '!'

Reply:
Yes, we generally do compare pointers to `!= nullptr`

Reply:
This was the second of two `nullptr` checks written this way. I just pushed a commit that changes this occurrence and the inverse check in the destructor at the top of this file. Please take a look again.